instrumentation/rocketmq/rocketmq-client/rocketmq-client-5.0/testing/src/main/java/io/opentelemetry/instrumentation/rocketmqclient/v5_0/RocketMqProxyContainer.java

```diff
@@ -6,34 +6,41 @@
 package io.opentelemetry.instrumentation.rocketmqclient.v5_0;
 
 import io.opentelemetry.instrumentation.test.utils.PortUtils;
+import org.testcontainers.DockerClientFactory;
 import org.testcontainers.containers.FixedHostPortGenericContainer;
 import org.testcontainers.containers.GenericContainer;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
 
-// TODO - Tests using this container will currently fail if TESTCONTAINERS_HOST_OVERRIDE is set to
-// anything other than "localhost".
 public class RocketMqProxyContainer {
   // TODO(aaron-ai): replace it by the official image.
-  private static final String IMAGE_NAME = "aaronai/rocketmq-proxy-it:v1.0.1";
+  private static final String IMAGE_NAME = "zackman0010/rocketmq-proxy-it:v1.0.2";
```

Comment:
pending https://github.com/aaron-ai/rocketmq-it/pull/1

Reply:
Done.

@zackman0010 . FYI, I have already pushed [aaronai/rocketmq-proxy-it:v1.0.2](https://hub.docker.com/layers/aaronai/rocketmq-proxy-it/v1.0.2/images/sha256-5ed2f5e1df44afce33f412d7f7ebe8fb6c6b586931f9b3cc38d4915d73042f42?context=repo) to dockerhub.

Reply:
Updated to the new IMAGE_NAME, tests are running now!